torchvision/transforms/autoaugment.py

```diff
@@ -249,7 +249,7 @@ class RandAugment(torch.nn.Module):
     `"RandAugment: Practical automated data augmentation with a reduced search space"
     <https://arxiv.org/abs/1909.13719>`_.
     If the image is torch Tensor, it should be of type torch.uint8, and it is expected
-    to have [..., 1 or 3, H, W] shape, where ... means an arbitrary number of leading dimensions.
+    to have [..., 3, H, W] shape, where ... means an arbitrary number of leading dimensions.
     If img is PIL Image, it is expected to be in mode "L" or "RGB".
```

Comment:
```suggestion
    If img is PIL Image, it is expected to be in mode "RGB".
```

Reply:
Oh, I forgot about changing the pillow part also. Thanks for pointing it.